packages/overlay/src/integrations/sentry/store/types.ts

```diff
@@ -14,7 +22,7 @@ export type Subscription = OnlineSubscription | EventSubscription | TraceSubscri
 
 export interface EventsSliceState {
   events: SentryEvent[];
-  eventIds: Set<string>;
+  eventsById: Map<string, SentryEvent>;
```

Comment:
this makes much more sense. from what I've heard you offline, we are going to go with maps for every id related event, right?

Reply:
yup, 100%